ui-tests/cypress/integration/administration/labels.spec.js

```diff
@@ -0,0 +1,46 @@
+import HomePage from '../../support/pageObjects/HomePage';
+import { htmlElements } from '../../support/pageObjects/WebElement';
+
+describe('Labels', () => {
+
+    let currentPage;
+
+    beforeEach(() => {
+        cy.wrap(null).as('labelToDelete');
+        cy.kcLogin('admin').as('tokens');
+    });
+
+    afterEach(() => {
+        cy.get('@labelToDelete').then((labelToDelete) => {
+            if (labelToDelete) {
+                cy.labelsController()
+                    .then(controller => controller.removeLabel(labelToDelete.code));
+            }
+        });
+        cy.kcLogout();
+    });
+
+    //SMOKE TEST
+    //ENG-3238
+    it('Labels section', () => {
```

Comment:
You can now rebase on develop and add proper tags instead of comments 😄

Reply:
Done